plugins/templates/index.js

```diff
@@ -53,7 +53,7 @@ exports.register = (server, options, next) => {
                     }
 
                     return user.getPermissions(pipeline.scmUri).then((permissions) => {
-                        if (!permissions[permission]) {
+                        if (!permissions[permission] && username in pipeline.admins === false) {
```

Comment:
This `permissions` returns from scm. Therefore, the user is an admin of a pipeline that publishes template, pemissions[admin] might be true. I think that this PR is not needed to fix issue.

Reply:
It was as pointed out. I close this PR.